### DISCUSSION TITLE
Docker not starting after upgrade to 3.4

### DISCUSSION THREAD

**Discussion by resparis** [netbox-community/netbox-docker#905] [Getting Help]
Hi All, 

I recently upgrade netbox-docker and I am having the following error.
etbox-docker_redis-cache_1 is up-to-date
Recreating netbox-docker_postgres_1 ... done
Recreating netbox-docker_netbox_1   ... done
Creating netbox-docker_netbox-worker_1 ... done

ERROR: for netbox-housekeeping  Container "246b67cc3b94" is unhealthy.
ERROR: Encountered errors while bringing up the project.

ubuntu@ip-10-10-50-172:~/projects/netbox-docker$ docker logs -f 246b67cc3b94
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/246b67cc3b94/json: dial unix /var/run/docker.sock: connect: permission denied

Any ideas?

**Comment by munterreitmaier** [comment]
Same Error for me.
Logs show that the database isn't ready after 30 sec.
